Kafka Samples that contain partition.assignment.strategy in properties file fails

This error occurs when using Kafka Consumer with a properties file that specifies `partition.assignment.strategy` incorrectly. Kafka changed the acceptable options, so some samples fail because they have "roundrobin," which is no longer supported. For the latest valid configuration, see here: http://kafka.apache.org/documentation.html#newconsumerconfigs

<pre>
Exception in thread "Thread-12" org.apache.kafka.common.KafkaException: Failed to construct kafka consumer
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:624)
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:518)
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:500)
	at com.ibm.streamsx.messaging.kafka.KafkaConsumerV9.<init>(KafkaConsumerV9.java:29)
	at com.ibm.streamsx.messaging.kafka.KafkaStringConsumerV9.<init>(KafkaStringConsumerV9.java:12)
	at com.ibm.streamsx.messaging.kafka.KafkaConsumerFactory.getClient(KafkaConsumerFactory.java:27)
	at com.ibm.streamsx.messaging.kafka.KafkaSource.allPortsReady(KafkaSource.java:116)
	at com.ibm.streams.operator.internal.runtime.api.OperatorAdapter.allPortsReady(OperatorAdapter.java:763)
	at com.ibm.streams.operator.internal.jni.JNIBridge.allPortsReady(JNIBridge.java:614)
Caused by: org.apache.kafka.common.KafkaException: roundrobin ClassNotFoundException exception occured
	at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:204)
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:566)
	... 8 more
Caused by: java.lang.ClassNotFoundException: roundrobin
	at java.lang.Class.forNameImpl(Native Method)
	at java.lang.Class.forName(Class.java:321)
	at org.apache.kafka.common.utils.Utils.newInstance(Utils.java:328)
	at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:202)
	... 9 more
</pre>

--- a/samples/KafkaSample/etc/consumer.properties
+++ b/samples/KafkaSample/etc/consumer.properties
@@ -1,6 +1,2 @@
 bootstrap.servers=broker.host.1:2181,broker.host.2:2181,broker.host.3:2181
-serializer.class=kafka.serializer.StringEncoder
 group.id=mygroup
-key.deserializer=org.apache.kafka.common.serialization.StringDeserializer
-value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
-partition.assignment.strategy=roundrobin

--- a/samples/KafkaSample/etc/producer.properties
+++ b/samples/KafkaSample/etc/producer.properties
@@ -1,4 +1,2 @@
 bootstrap.servers=broker.host.1:2181,broker.host.2:2181,broker.host.3:2181
 acks=0
-key.serializer=org.apache.kafka.common.serialization.StringSerializer
-value.serializer=org.apache.kafka.common.serialization.StringSerializer

